packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -108,6 +108,11 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('unlockCaptureOrientation() is not implemented.');
   }
 
+  @override
+  Stream<CameraTimeLimitReachedEvent> onCameraTimeLimitReached(int cameraId) {
+    throw UnimplementedError('onCameraTimeLimitReached() is not implemented.');
+  }
```

Comment:
This change to the platform interface should probably be split off to a separate PR, so that it can be published to pub.dev before the rest is merged to master.

Reply:
I think it would be better to rename this to `onVideoRecordingTimeLimitReached`, so it is clear this is directly related to the video recording feature. Of course if you accept my earlier comment about also emitting the recorded video on this stream when the user calls stopVideoRecording we should rename it to something more generic like `onVideoRecorded`.